docs/src/index.rst

```diff
@@ -30,9 +30,9 @@
 
    tutorials/scikit-learn
    tutorials/pytorch-mnist
+   tutorials/cluster
```

Comment:
I agree it's more important but there seems to be an implicit categorization. Like specific applications vs ways to use it. Do you think we should make it explicit and making subsections in the tutorial menu?

Reply:
I don't think it's worth it to do it at this time because there is not that many tutorials in total and because the two tutorials after *cluster* are very specific case.